File: Q_760.cpp
class Solution {
public:
    vector<int> anagramMappings(vector<int>& A, vector<int>& B) {
        map<int,int> mymap;
        vector<int> ret;
        int i;
        for(i=0;i<B.size();i++)
            mymap[B[i]]=i;
        for(i=0;i<A.size();i++)
            ret.push_back(mymap[A[i]]);
        return ret;
    }
};

File: python/Q_760.py
class Solution:
    def anagramMappings(self, A, B):
        """
        :type A: List[int]
        :type B: List[int]
        :rtype: List[int]
        """
        mymap = {}
        mylist=[]
        j=0
        for i in B:
            mymap[i]=j
            j=j+1
        for i in A:
            mylist.append(mymap[i]) 
        return mylist
